main.cc

```diff
@@ -648,7 +648,8 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
     init("list-tools", bpo::bool_switch(), "list included tools and exit");
 
     bpo::options_description deprecated_options("Deprecated options - ignored");
-    cfg->add_deprecated_options(deprecated_options.add_options());
+    auto deprecated_options_easy_init = deprecated_options.add_options();
+    cfg->add_deprecated_options(deprecated_options_easy_init);
```

Comment:
BTW, we never use the return values of `config_file::add_options()` and `config_file::add_deprecated_options()`, as we are not chaining calls to add more options to the `options_description_easy_init` instance by these functions.